lib/vega_lite/convert/native.ex

```diff
@@ -0,0 +1,31 @@
+defmodule VegaLite.Native do
+  @moduledoc false
+
+  mix_config = Mix.Project.config()
+  version = mix_config[:version]
+  github_url = mix_config[:package][:links]["GitHub"]
+
+  use RustlerPrecompiled,
+    otp_app: :vega_lite_convert,
+    crate: "ex_vl_convert",
+    version: version,
+    base_url: "#{github_url}/releases/download/v#{version}",
+    targets:
+      Enum.uniq(["aarch64-unknown-linux-musl" | RustlerPrecompiled.Config.default_targets()]),
+    force_build: System.get_env("VEGA_LITE_CONVERT_BUILD") in ["1", "true"]
+
+  # Vega related NIF functions
+  def vega_to_html(_vg_json_spec, _bundle, _renderer), do: :erlang.nif_error(:nif_not_loaded)
+  def vega_to_jpeg(_vg_json_spec, _scale, _quality), do: :erlang.nif_error(:nif_not_loaded)
+  def vega_to_pdf(_vg_json_spec), do: :erlang.nif_error(:nif_not_loaded)
+  def vega_to_png(_vg_json_spec, _scale, _ppi), do: :erlang.nif_error(:nif_not_loaded)
+  def vega_to_svg(_vg_json_spec), do: :erlang.nif_error(:nif_not_loaded)
```

Comment:
I would skip these, since we don't use them.

Reply:
I was actually hoping to use these in an upcoming project where I need a bit more fine grained control over visuals beyond what VegaLite exposes. Any chance we could leave them in?

Reply:
I'm ok leaving these as is, but obviously without any compatibility guarantee. Is the project an app or a library? How are you planning to use it, build Vega JSON by hand?

Reply:
Correct. I plan on creating the JSON definitions by hand and then passing through to these functions. This will be part of an app versus a library....for now at least.

Reply:
Fine by me, unless @josevalim has objections :D

Reply:
Ship it!